LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014, NFLabs inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
I checked some TLPs to see how they do. And looks like it's enough. I'm also curious if it is enough (and why).